pages/Documents/MessagingChannels/PushNotificationService/overview.md

```diff
@@ -21,14 +21,5 @@ Mobile App Messaging provides brands with the ability to provide their customers
 
 ![PushServices](img/pushservices.png)
 
-LivePerson’s push service interfaces directly with Apple’s APNS and Google’s FCM push notification services. For brands who wish to refrain from sharing their tokens/certificates, LivePerson offers the option to configure their push to the URIs of their choice and to utilize their own logic.
-
-### Getting Started
-
-1. **Retrieve your domain**. Use the [LivePerson Domain API](agent-domain-domain-api.html) to retrieve this information by providing the following service name:
-
-	* pusher
-
-3. Note the [API terms of use](https://www.liveperson.com/policies/apitou).
-
-4. When using this API, it is recommended that you implement our [Retry Policy and KeepAlive best practices](guides-retry-policy.html).
+LivePerson’s push service interfaces directly with Apple’s APNS and Google’s FCM push notification services. For brands who wish to refrain from sharing their tokens/certificates, LivePerson offers the option to configure their push to the URIs of their choice and to utilize their own logic, see:
+[Push Proxy](Configuration/push-proxy.md)
```

Comment:
Please remove the line break, or add another line break.

Also, please check on the link.